Mathlib/Data/Finset/Lattice/Fold.lean

```diff
@@ -102,8 +102,8 @@ theorem sup_const_le : (s.sup fun _ => a) ≤ a :=
 theorem le_sup {b : β} (hb : b ∈ s) : f b ≤ s.sup f :=
   Finset.sup_le_iff.1 le_rfl _ hb
 
-theorem isLUB_sup (s : Finset α) : IsLUB s (sup s id) :=
-  ⟨fun x h => id_eq x ▸ le_sup h, fun _ h => Finset.sup_le h⟩
+lemma isLUB_sup : IsLUB (f '' s) (s.sup f) := by
```

Comment:
Let's keep the old one around since it has a convenient defeq, and `s.sup id` does show up often in practice. `isLUB_sup_id`?